x-pack/plugins/security/server/routes/api_keys/enabled.ts

```diff
@@ -8,15 +8,18 @@ import { wrapIntoCustomErrorResponse } from '../../errors';
 import { createLicensedRouteHandler } from '../licensed_route_handler';
 import { RouteDefinitionParams } from '..';
 
-export function defineEnabledApiKeysRoutes({ router, authc }: RouteDefinitionParams) {
+export function defineEnabledApiKeysRoutes({
+  router,
+  getAuthenticationService,
+}: RouteDefinitionParams) {
   router.get(
     {
       path: '/internal/security/api_key/_enabled',
       validate: false,
     },
     createLicensedRouteHandler(async (context, request, response) => {
       try {
-        const apiKeysEnabled = await authc.areAPIKeysEnabled();
+        const apiKeysEnabled = await getAuthenticationService().apiKeys.areAPIKeysEnabled();
```

Comment:
**note to self** I'd like to keep an eye on external usage of the authentication service within routes. This might be a good candidate to become an http context provider in the future.

I don't mind our plugins having to play this setup/start service juggling for our own internal services, but if this ends up being used in more places outside of the security plugin, then it might make sense to make it easier to consume

Reply:
Yep, that's a good idea